### PR TITLE
DerpQuest: Update Chinese translations

### PR DESCRIPTION
**Pull request by ALKALiKong233**

### CODE REVIEW

**Review by idoybh — APPROVED** [review]

File: res/values-zh-rCN/derpquest_strings.xml [hunk 1]
```diff
@@ -378,6 +379,11 @@
     <string name="status_bar_battery_percentage_text_next">图标旁</string>
     <string name="status_bar_battery_text_charging_title">充电时电池百分比</string>
     <string name="status_bar_battery_text_charging_summary">充电时始终显示电池百分比</string>
+    <string name="status_bar_battery_charging_bolt_title">显示充电符号</string>
+    <string name="status_bar_battery_charging_bolt_summary">充电时在电量文本旁显示一个 \u26A1\uFE0E 符号</string>
+
+    <!-- QS settings categories -->
+    <string name="category_tiles_title">Tiles area</string>
```
Comment (idoybh) [inline res/values-zh-rCN/derpquest_strings.xml]:
```suggestion
```
It's literally already defined one line below. Please. Check your file next time. Try to compile with it at least!

**Review by idoybh — CHANGES_REQUESTED** [review]

File: res/values-zh-rCN/derpquest_strings.xml (hunk 1 above)
Comment (idoybh) [reply res/values-zh-rCN/derpquest_strings.xml]: Please force push to this branch with the requested change. You disallowed me from editing your changes

**Comment by ALKALiKong233** [comment]
Finished. Is it OK now?

**Review by idoybh — APPROVED** [review]

**Comment by idoybh** [comment]
thank you :)

**Pull request merged**